Lib/test/test_sys_settrace.py

```diff
@@ -3146,5 +3146,37 @@ def func(arg = 1):
         sys.settrace(None)
 
 
+class TestTraceAtRecursionLimit(unittest.TestCase):
+    def setUp(self):
+        self.addCleanup(sys.settrace, sys.gettrace())
+
+    def test_does_not_crash(self):
+        # Moved from Lib/test/crashers as it does not crash anymore.
+        # This test ensures that the code does not start crashing again.
+        # Original comment:
+
+        # From http://bugs.python.org/issue6717
+
+        # A misbehaving trace hook can trigger a segfault by exceeding the recursion
+        # limit.
+        def x():
+            pass
+
+        def g(*args):
+            if True: # change to True to crash interpreter
```

Comment:
This comment seems wrong.

Reply:
It does, but I copied it verbatim from the original crasher. I can remove it though if you prefer?